crates/oxc_linter/src/autofix/mod.rs

```diff
@@ -5,26 +5,13 @@ mod fixer;
 pub use fixer::Fixer;
 
 #[derive(Debug)]
-pub struct Fix {
-    pub content: String,
+pub struct Fix<'a> {
+    pub content: &'a str,
```

Comment:
I wonder whether this should be a `Cow`

Reply:
But I think that the `content` doesn't need to be a owned data, can't we just store a `str` reference?

Reply:
Code fixes are mostly allocated strings, they are rarely a string reference.
Remember, you will be creating the fix inside a linter rule.

Reply:
What about using trait bound? Something like `T: Into<Cow<'a, str>>`, such that the user can create a fix with a string reference or a `String`.

Reply:
You are right. The rule may use `String` to create a fix, so we need to store a `Cow` instead of just a string reference.